api/src/test/java/uk/gov/dhsc/htbhf/dwp/service/EligibilityServiceTest.java

```diff
@@ -33,18 +42,64 @@
     @MockBean
     private RestTemplate restTemplate;
 
+    @MockBean
+    private UCHouseholdRepository ucHouseholdRepository;
+
+    @MockBean
+    private LegacyHouseholdRepository legacyHouseholdRepository;
+
     @Autowired
     private EligibilityService eligibilityService;
 
     @Test
-    void shouldCreateEligibilityRequest() {
+    void shouldReturnResponseFromUCDatabaseIfFound() {
+        EligibilityRequest eligibilityRequest = anEligibilityRequest();
+        UCHousehold household = aHousehold();
+        given(ucHouseholdRepository.findHouseholdByAdultWithNino(anyString())).willReturn(Optional.of(household));
+
+        var response = eligibilityService.checkEligibility(eligibilityRequest);
+
+        assertThat(response.getNumberOfChildrenUnderFour()).isEqualTo(2);
+        assertThat(response.getNumberOfChildrenUnderOne()).isEqualTo(1);
+        assertThat(response.getHouseholdIdentifier()).isEqualTo(household.getHouseholdIdentifier());
+        assertThat(response.getEarningsThresholdExceeded()).isEqualTo(household.getEarningsThresholdExceeded());
+
+        verify(ucHouseholdRepository).findHouseholdByAdultWithNino(eligibilityRequest.getPerson().getNino());
+        verify(legacyHouseholdRepository, never()).findHouseholdByAdultWithNino(anyString());
+        verify(restTemplate, never()).postForEntity(anyString(), any(), any());
```

Comment:
I find its clearer to use `verifyZeroInteractions(restTemplate)`, it also makes sure that its not called at all.

Reply:
Done.